warehouse/query-core/src/main/java/org/apache/commons/jexl2/parser/JexlNodes.java

```diff
@@ -175,6 +175,29 @@ public static void setLiteral(ASTStringLiteral literal, String value) {
         literal.image = value;
     }
     
+    /**
+     * Null out parent and child references for each of the provided nodes
+     *
+     * @param nodes
+     *            a JexlNode collection
+     */
+    public static void blank(JexlNode... nodes) {
+        for (JexlNode node : nodes) {
+            blank(node);
+        }
+    }
+    
+    /**
+     * Null out parent and child references
+     *
+     * @param node
+     *            a JexlNode
+     */
+    public static void blank(JexlNode node) {
+        node.parent = null;
+        node.children = new JexlNode[0];
```

Comment:
This seems unnecessary to me, unless there's some functional reason to do this that I'm missing.

Reply:
After some digging I don't think this is necessary. This could be a replaced with a null check if you think eliminating pointers is a good idea, otherwise I'll just remove these methods.

Reply:
My preference would be to just remove the methods.

Reply:
Sounds good, I'll remove the methods